jetty-server/src/main/java/org/eclipse/jetty/server/handler/gzip/GzipHandler.java

```diff
@@ -156,7 +155,7 @@
 {
     public static final String GZIP = "gzip";
     public static final String DEFLATE = "deflate";
-    public static final int DEFAULT_MIN_GZIP_SIZE = 16;
+    public static final int DEFAULT_MIN_GZIP_SIZE = 23;
```

Comment:
While 23 is the size break even point, the CPU break even point is probably a lot higher.  I'd be OK with us setting this to be something like 1024, which is close to a MTU, so sending a single TCP frame can often be about the same cost regardless of size.